sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/SchemaPruningSuite.scala

```diff
@@ -774,4 +774,17 @@ abstract class SchemaPruningSuite
         assert(scanSchema === expectedScanSchema)
     }
   }
+
+  testSchemaPruning("extract case-insensitive struct field from array") {
```

Comment:
Do we need to have a test coverage for `extract case-insensitive struct field` too?

Reply:
It is fine (https://github.com/apache/spark/pull/32059#discussion_r608277894), but it is better to add a test for better coverage too. Let me add one.